conda_build/pypi.py

```diff
@@ -454,236 +454,188 @@ def version_compare(args, package, versions):
 
 
 def get_package_metadata(args, package, d, data):
-    # Unfortunately, two important pieces of metadata are only stored in
-    # the package itself: the dependencies, and the entry points (if the
-    # package uses distribute).  Our strategy is to download the package
-    # and "fake" distribute/setuptools's setup() function to get this
-    # information from setup.py. If this sounds evil, keep in mind that
-    # distribute itself already works by monkeypatching distutils.
 
-    import yaml
     print("Downloading %s (use --no-download to skip this step)" % package)
-    tempdir = mkdtemp('conda_skeleton_' + d['filename'])
 
     [output_dir] = args.output_dir
 
-    if not isdir(SRC_CACHE):
-        makedirs(SRC_CACHE)
-
-    try:
```

Comment:
@asmeurer This `try` block was used only to `rm_rf` the temporary directory in the `finally` block at [ln685](https://github.com/conda/conda-build/pull/449/files#diff-06936d397c05412b6dd38c5393523f72L685). This block has now been moved to the `get_pkginfo` function. Removing the `try` reduced the indentation of the whole code by one letter and this is leading to this shitty diff. To get a more useful diff use the `--historgram` algorithm, unfortunately github's diffs don't have option to change the diff algorithm used.